Plugins/NativeInput/Source/NativeInput.cpp

```diff
@@ -85,7 +85,7 @@ namespace Babylon::Plugins
             SetInputState(DeviceType::Touch, pointerId, inputIndex, 0, deviceInputs);
 
             // If all "buttons" are up, then remove the device (e.g. device "disconnected").
-            for (int32_t index = 0; index < deviceInputs.size(); index++)
+            for (uint index = 0; index < deviceInputs.size(); index++)
```

Comment:
Why did you need to make this change? Also should it be `uint_32t`?

Reply:
When I brought the code into BabylonReactNative it threw this as a warning.  Since we have warnings as errors turned on this was a build break only in the context of BabylonReactNative.  The type for vector.size seems to be size_type, which is dependent on the allocator, but is always unsigned.  Looks like it usually defaults to size_t so I'll use that instead.